frameworks/person-escort-record/questions/property-bag-type.yml

```diff
@@ -18,6 +18,7 @@ options:
       hint: For example, 23.99
       display:
         character_width: 5
+      prefix: £
```

Comment:
this seems to be in favour of https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/pull/20
which we can safely close now